docs/plugins/bootstrap-vue.js

```diff
@@ -1,4 +1,5 @@
 import Vue from 'vue'
 import BootstrapVue from '../../src'
+import '../../dist/bootstrap-vue.css'
```

Comment:
This requires you to have to run `dist` before funning `docs-dev`.

Reply:
Yeah, moved it to the nuxt-config.js file

Reply:
Actually, will probably need to get nuxt to use a loader for the SCSS